src/main/kotlin/org/jitsi/nlj/MediaSourceDesc.kt

```diff
@@ -0,0 +1,161 @@
+/*
+ * Copyright @ 2018 - present 8x8, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jitsi.nlj
+
+import java.util.Arrays
+import org.jitsi.nlj.RtpLayerDesc.Companion.getEncodingId
+import org.jitsi.nlj.rtp.VideoRtpPacket
+import org.jitsi.utils.ArrayUtils
+
+/**
+ * Represents a collection of [RtpLayerDesc]s that encode the same
+ * media source. This specific implementation provides webrtc simulcast stream
+ * suspension detection.
```

Comment:
I think at one point you mentioned clarifying the definition of a source in the docs (and putting a reference to the RFC defining 'source' I think?)